src/slices/ShopCheckout/ShopCheckout.tsx

```diff
@@ -160,76 +165,107 @@ export const ShopCheckout = ({ slice }: ShopCheckoutProps): JSX.Element => {
+                            style: 'unit',
+                            unit: 'kilogram',
+                            maximumFractionDigits: 0,
                           }
-                          errorMessage={errors.contributionValue}
-                        />
-                      )}
-                    </Field>
-                  </Box>
-
-                  <Spacer height="6" />
-
-                  <SimpleGrid columns={2} gap="4">
-                    <LabelNumberPair
-                      label={formatMessage({
-                        id: 'sections.shopCheckout.summary.kg',
-                      })}
-                      number={`${formatNumber(
-                        Math.floor(values.contributionValue / slice.pricePerKg),
-                        {
-                          style: 'unit',
-                          unit: 'kilogram',
-                          maximumFractionDigits: 0,
-                        }
-                      )} CO₂`}
-                    />
-                    <LabelNumberPair
-                      label={formatMessage({
-                        id: 'sections.shopCheckout.summary.price',
-                      })}
-                      number={formatNumber(
-                        Math.floor(
-                          values.contributionValue / slice.pricePerKg
-                        ) * slice.pricePerKg,
-                        {
-                          style: 'currency',
-                          currency: slice.currency,
-                          minimumFractionDigits: 2,
-                          maximumFractionDigits: 2,
+                        )} CO₂`}
+                      />
+                      <LabelNumberPair
+                        label={formatMessage({
+                          id: 'sections.shopCheckout.summary.price',
+                        })}
+                        number={formatNumber(
+                          Math.floor(
+                            values.contributionValue / slice.pricePerKg
+                          ) * slice.pricePerKg,
+                          {
+                            style: 'currency',
+                            currency: slice.currency,
+                            minimumFractionDigits: 2,
+                            maximumFractionDigits: 2,
+                          }
+                        )}
+                        caption={
+                          slice.taxInPercent &&
+                          slice.taxInPercent > 0 &&
+                          summaryPrice > 0
+                            ? formatMessage(
+                                {
+                                  id: 'sections.shopCheckout.summary.price.taxIncluded',
+                                },
+                                {
+                                  number: formatNumber(
+                                    summaryPrice +
+                                      (summaryPrice * slice.taxInPercent) / 100,
```

Comment:
```suggestion
                                      summaryPrice * (slice.taxInPercent / 100),
```
I think it makes more sense like this regarding readibility